chore(http): Refresh headers docs i-r (#36079)
mdn/content: files/en-us/web/http/headers/no-vary-search/index.md

## Directives

- `key-order`
  - : If included in the header value, it indicates that differences in the order of parameters between otherwise identical URLs will not cause them to be cached as separate entries. Differences in the parameters present _will_ cause them to be cached separately.
- `params`

>   - : Either a boolean (as a literal `params`) or a list of strings:
>     - If included in the header value as a boolean, it indicates that differences in parameters between otherwise identical URLs will not cause them to be cached as separate entries.
>     - If included in the header value as a list, it indicates that the presence of the specific parameters listed will not cause otherwise identical URLs to be cached as separate entries. The presence of other parameters _will_ cause them to be cached separately.

Reviewer: Not sure what the params with string would look like (second case below) ???

```http
No-Vary-Search: params
No-Vary-Search: param1 param2 param3
```

Is a list comma separated? Does this need a prefix - i.e. No-Vary-Search: params param1 param2 param3 ?

Anyway, a wording more like this might be better:

```suggestion
  - : Either the keyword `params` or a list of strings:
    - If the literal `params` is specified, it indicates that differences in parameters between otherwise identical URLs will not cause them to be cached as separate entries.
    - If a list of params are included in the header, it indicates that the presence of the specific parameters listed will not cause otherwise identical URLs to be cached as separate entries.
      The presence of other parameters _will_ cause them to be cached separately.
```

Reviewer: I think so!
